crates/bevy_reflect/bevy_reflect_derive/src/utility.rs

```diff
@@ -29,6 +29,13 @@ pub(crate) struct ResultSifter<T> {
     errors: Option<syn::Error>,
 }
 
+pub(crate) fn field_ident_or_indexed(index: usize, ident: Option<&Ident>) -> Member {
```

Comment:
Awesome! Could we add a simple doc comment to this that explains what it's meant to do?

Also, (nit) what do you think about the name `get_field_member` or something similar?